boards/DCM/Src/App/App_SetPeriodicCanSignals.c

```diff
@@ -0,0 +1,87 @@
+#include "App_SharedSetPeriodicCanSignals.h"
+#include "App_SetPeriodicCanSignals.h"
+
+//This generates an error and doesn't build: ISO C forbids forward references to 'enum' types [-Werror=pedantic]
+// Why?
+STATIC_DEFINE_APP_SET_PERIODIC_CAN_SIGNALS_IN_RANGE_CHECK(DcmCanTxInterface)
```

Comment:
Did you include App_InRangeCheck.h?

Reply:
I did not, that fixed it

Reply:
Sounds good. Don't forget to remove the comment above.

Reply:
removed